contentreview-impl/pack/src/webapp/WEB-INF/components.xml

```diff
@@ -63,14 +65,15 @@
     </bean>
 
 	<!--  Uncomment this to use a site property to define which sites use c-r -->
-	<!--
-		<bean id="org.sakaiproject.contentreview.service.ContentReviewSiteAdvisor" class="org.sakaiproject.contentreview.impl.adivisors.SitePropertyAdvisor">
+	

```

Comment:
You have 2 `org.sakaiproject.contentreview.service.ContentReviewSiteAdvisor` defined in here.